tools/manifest/sourcefile.py

```diff
@@ -209,7 +209,7 @@ def __init__(self, tests_root, rel_path, url_base, hash=None, contents=None):
         if os.name == "nt":
             # do slash normalization on Windows
             if isinstance(rel_path, binary_type):
-                rel_path = rel_path.replace(b"/", b"\\")
+                rel_path = rel_path.replace("/", "\\")
```

Comment:
These two codepaths now look to do the same thing on Python 3. Is one actually unused?

Reply:
I guess you're right